src/components/LocationWeather/LocationWeather.jsx

```diff
@@ -31,6 +31,7 @@ class LocationWeather extends PureComponent {
         <LocationWeatherCurrentInfo
           currentLocationWeather={this.props.currentLocationWeather}
           currentLocationInfo={this.props.currentLocationInfo}
+          addToSelectedLocations={this.props.addToSelectedLocations}
```

Comment:
how about removal? would make more sense to have `onChangeLocationSelection={this.props....}` ?